### ISSUE TITLE
Publish VA debt

### ISSUE THREAD

**Issue by aprocik1** [department-of-veterans-affairs/va.gov-team#78588]
## Description
First, we need to update this modern page with stakeholder feedback:

- [How your reason for withdrawing from a class affects your VA debt](https://www.va.gov/resources/how-your-reason-for-withdrawing-from-a-class-affects-your-va-debt/)

Then, we need to redirect a legacy URL to a modern URL. And we need to update the link destination and text from the legacy URL to the modern URL in the "more information and resources" section on this modern page:

- [VA education and training benefits](https://www.va.gov/education/)

### Step 1: Update the VA debt page
- [x] DB: Update the page in Drupal with feedback from stakeholders
- [x] AP: Review update
- [x] AP: Share update with BP
- [x] BP: Review update
- [x] DB: Share update with stakeholders
- [x] Stakeholders: Review update
~~- [ ] DB: Share update with MZ and ST~~
~~- [ ] MZ, ST: Copyedit and publish~~

### Step 2: Redirect URLs
- [ ] DB: Work with IA to open a redirect ticket

Redirect this legacy URL:

- [Information About GI Bill Overpayments and Debts](https://www.benefits.va.gov/gibill/resources/education_resources/debt_info.asp)

To this modern URL:

- [How your reason for withdrawing from a class affects your VA debt](https://www.va.gov/resources/how-your-reason-for-withdrawing-from-a-class-affects-your-va-debt/)

### Step 3: Update the VA education and training URL
- [ ] DB: Update the link destination and text in Drupal
- [ ] AP: Review update
- [ ] AP: Share update with BP
- [ ] BP: Review update
- [ ] MZ, ST: Copyedit and publish

## Criteria
- [ ] Published updated page with stakeholder feedback
- [ ] IA opened a redirect ticket for the legacy URL
- [ ] Published updated link destination and text on modern page

**Comment by aprocik1** [comment]
@d-beyer, I just updated this ticket. Here are some considerations as you work on step 1:

- We should aim to keep our page introduction to a few sentences. Since the stakeholders want to expand that content, we may need to move it out of the page introduction and into a new H2.
- On resources and support pages, I don't think we can place an action link in the middle of a page introduction. I think our only option in Drupal is to place it underneath.
- The language "leave school" doesn't feel clear and specific enough. We may need to use different language, like "withdraw from (or “drop”) a class _or program_." But even that may not capture the full meaning of "leave school."

Let me know if you any questions as you get into the work. Thanks for moving this forward!

**Comment by d-beyer** [comment]
@aprocik1 I drafted the changes to the in Drupal: [How your reason for withdrawing from a class affects your VA debt - Revision
](https://preview-prod.vfs.va.gov/preview?nodeId=33833)

Since this page uses reusable Q&As, I also had to update [What are mitigating circumstances?](https://prod.cms.va.gov/node/33830/latest). The change here won't appear in the preview of the update page. I also had to remove the second mention of "a class" because I was right up against the character limit.

The _new_ Q&A I made appears on the draft, but here's the URL for that as well: [Q&A: What if the school refunds less than I what owe?](https://prod.cms.va.gov/resources/what-if-the-school-refunds-less-than-i-what-owe).

Let me know what you think. Happy to do some real-time workshopping since I don't feel supremely confident in the school/program/class thing. Or we could ask Beth or Tammy for clarification.

**Comment by aprocik1** [comment]
@d-beyer, thanks! I made these small edits:

- [How your reason for withdrawing from a class affects your VA debt](https://prod.cms.va.gov/node/33833/revisions/view/854230/856118/visual_inline)
- [What are mitigating circumstances?](https://prod.cms.va.gov/node/33830/revisions/view/854238/856105/visual_inline)
- [What if the school doesn't pay back the full amount it owes?](https://prod.cms.va.gov/node/66908/revisions/view/854228/856116/visual_inline)

Can you summarize your edits to this page in a few bullet points - based on the feedback from stakeholders? We'll need to explain our decisions to accept or reject their feedback.

**Comment by d-beyer** [comment]
@aprocik1 Sure thing, here ya go.

> We edited a few things from your  proposed changes:
> - You added some text to the introduction of the page. Introductions on VA.gov pages are kept quite short and don't contain much instruction or explanation, so we moved this text to its own Q&A section lower on the page. It has the header "What if the school doesn't pay back the full amount it owes?"
> - You added the phrase "leave school" in a few places in the text. We've added mentions of withdrawing from programs and schools, but didn't use the word "leave". Please let us know if that doesn't convey the full meaning of your change.
> - We made some other very light editing to the text provided that doesn't change its meaning, primarily in the introduction and the new Q&A.

**Comment by d-beyer** [comment]
Update: I shared the revised page information with stakeholders on 4/2.

**Comment by d-beyer** [comment]
Update from 05.01 meeting with SMEs:
- SMEs called out some issues, including the fact that books and supplies can't be "debted"
- However, we may need the SMEs to review this in full.

**Comment by d-beyer** [comment]
Update: We met with SMEs today and [rough-revised the text live](https://dvagov.sharepoint.com/:w:/r/sites/SitewideCAIA/Shared%20Documents/1.0%20Migration%20work/Education%20service/Revision%20-%20How%20your%20reason%20for%20withdrawing%20from%20a%20class%20affects%20your%20VA%20debt.docx?d=wff3bb0fb05fe42b79ffe06fba55dc629&csf=1&web=1&e=xzGiuI). I'm going to incorporate their feedback, then send them a screenshot/preview link for final approval.

**Comment by d-beyer** [comment]
At Beth's direction, I implemented the changes given to us by the SMEs in Drupal and have handed off the rest of the task to her to get it across the finish line.

The updates live in these drupal pages.
- [How your reason for withdrawing from a class affects your VA debt](https://prod.cms.va.gov/resources/how-your-reason-for-withdrawing-from-a-class-affects-your-va-debt)
- [What are mitigating circumstances?](https://prod.cms.va.gov/resources/what-are-mitigating-circumstances)
- [How do I report mitigating circumstances?](https://prod.cms.va.gov/resources/how-do-i-report-mitigating-circumstances)
- [What’s the 6-credit-hour exclusion?](https://prod.cms.va.gov/resources/whats-the-6-credit-hour-exclusion)

**Comment by aprocik1** [comment]
Update: Beth told Derek that we're done with our portion off the migration work on this page. Beth will work with SMEs to make any remaining updates to the page, and update the link destination on this page: [VA education and training benefits](https://www.va.gov/education/). Derek requested the redirect from Jenny, and Jenny will follow up with SMEs to implement it. I'm closing this ticket as complete.
